### DISCUSSION TITLE
Will a repository get flagged if a lot of people star it in a short period of time?

### DISCUSSION THREAD

**Discussion by jameslimjy** [community/community#102828] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi all, wanted to check something regarding star-ing a repository. 

If I have a campaign that involves a large number of individuals (~1000 accounts) star-ing a repository in a short period of time (1 hour), will there be any issues? For e.g., will that trigger any sybil-resistance protection on Github's end that'll cause the repository to be flagged?

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @jameslimjy! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, accounts are typically flagged if they violate either our [Community Code of Conduct](https://github.com/community/community/blob/0b1c28ac32f8ebaf8048fa512f466c24fdfa17b2/CODE_OF_CONDUCT.md) or [GitHub's Terms of Service](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service). I recommend reviewing both and making sure that your activity does not violate any of the terms listed there. 

I've gone ahead and moved this to the correct category for you. Good luck!
